File: CompSci 1/Unit 12, Selection/GUI02/GUI02.java
import java.awt.*;
import javax.swing.*;

public class GUI02 extends JFrame
{
    // Step 1 : create JLabel variable
    private JLabel label;

    // constructor
    public GUI02()
    {
        // Step 2 : create JLabel object and store its reference in label
        label = new JLabel("Have a nice day!");

        // Step 3 : set label attributes
        label.setLocation(150, 200);
        label.setSize(200, 20);
        label.setForeground(Color.RED);
        label.setFont(new Font("Arial", Font.PLAIN, 24));

        // Step 4: add label to content pane of frame
        getContentPane().add(label);

        // set frame attributes
        setLayout(null);
        setSize(500, 500);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        getContentPane().setBackground(Color.white);
        setVisible(true);
    }

    // main method
    public static void main(String[] args)
    {
        GUI02 app = new GUI02();   // run program
    }
}   

File: CompSci 1/Unit 12, Selection/GUI02/GUI02.kt
import java.util.*
import java.awt.*
import javax.swing.*
import java.awt.event.*

class GUI02 : JFrame() {
    // Step 1 : create JLabel variable
    private val label: JLabel

    // constructor
    init {
        // Step 2 : create JLabel object and store its reference in label
        label = JLabel("Have a nice day!")

        // Step 3 : set label attributes
        label.setLocation(150, 200)
        label.setSize(200, 20)
        label.foreground = Color.RED
        label.font = Font("Arial", Font.PLAIN, 24)

        // Step 4: add label to content pane of frame
        contentPane.add(label)

        // set frame attributes
        layout = null
        setSize(500, 500)
        defaultCloseOperation = JFrame.EXIT_ON_CLOSE
        contentPane.background = Color.white
        isVisible = true
    }
}

fun main(args: Array<String>) {
    val app = GUI02()   // run program
}